### PR TITLE
fix image fields

### PR DESCRIPTION
**Pull request by kimdhamilton**
The 2 image fields changed in v2 can be changed back to v1 version: image:certificate and image:logo can be image, as they were in v1. The reason they were failing validation is because the OBI spec allows these to be 'oneOf' url or data uri -- the content we provide for some reason validates against both. I fixed cert-schema to override OBI's spec, and check 'anyOf', which passes, and allows is to:
- improve OBI compliance
- minimize v1/v2 changes

I'm following up with OBI folks on this.

Confirmed unit tests pass. It looks like cert-web-component is not affected by this (based on code search).

### CODE REVIEW

File: Models/Certificate.swift [hunk 1]
```diff
@@ -338,23 +338,7 @@ private struct CertificateV1_1 : Certificate {
...
-                return nil
-        }
-        let logo = imageData(from: logoURI)
-        
-        return Issuer(name: issuerName,
-                      email: issuerEmail,
-                      image: logo,
-                      id: issuerIdURL,
-                      url: issuerURL)
+        return MethodsForV1_1.parse(issuerJSON: issuerJSON)
```
Comment (kimdhamilton) [inline Models/Certificate.swift:341]: now there are no differences between issuer v1/2 parsing.

Comment (bedinotti) [reply Models/Certificate.swift:341]: That's great! 👍

File: cert-walletTests/Resources/sample_signed_cert-1.2.0.json [hunk 2]
```diff
@@ -7,14 +7,14 @@
...
-            "image:certificate": "data:image/png;base64,",
+            "image": "data:image/png;base64,",
             "id": "https://www.theissuer.edu/criteria/2016/05/certificate-type.json",
             "issuer": {
                 "@type": "Issuer",
                 "url": "http://www.theissuer.edu",
                 "name": "Issuing Institution",
                 "id": "https://www.theissuer.edu/issuer/the-issuer.json",
-                "image:logo": "data:image/png;base64,",
+                "image": "data:image/png;base64,",
```
Comment (bedinotti) [inline cert-walletTests/Resources/sample_signed_cert-1.2.0.json:17]: Doesn't making changes to a signed cert mean it should now fail validation?

File: cert-walletTests/Resources/sample_signed_cert-valid-1.2.0.json [hunk 3]
```diff
@@ -1 +1 @@
...
...rzrELABKmCzZ16QcAE6UXTnmcVAiZIFWzu1AsCJkgvmvY8qxAwQapgc6deEDBBetG051mFgAlSBZs79YKACdKLpj3PKgRMkCrY3KkXBEyQXjTteVYhYIJUweZOvSBggvSiac+zCgETpAo2d+oFAROkF017nlUImCBVsLlTLwiYIL1o2vOsQsAEqYLNnXpBwATpRdOeZxUCJkgVbO7UCwImSC+a9jyrEDBBqmBzp14QMEF60bTnWYWACVIFmzv1goAJ0oumPc8qBEyQKtjcqRcETJBeNO15ViFgglTB5k69IGCC9KJpz7MKAROkCjZ36gUBE6QXTXueVQiYIFWwuVMvCJggvWja86xCwASpgs2dekHABOlF055nFQImSBVs7tQLAiZIL5r2PKsQ+H/qKJnesbOAFwAAAABJRU5ErkJggg==", "language": "en-US", "id": "http://certificates.gamoeofthronesxyz.org/criteria/2016/08/got.json", "description": "This certifies that the named character is an official Game of Thrones character.", "title": "Game of Thrones Character"}}, "@context": "https://w3id.org/blockcerts/context#", "@type": "BlockchainCertificate", "receipt": {"proof": [{"left": "7dd64b40a24d5874e4024e0020456835125cd2040805b9da7b30af38a5ecfa16"}], "@type": "BlockchainReceipt", "merkleRoot": "40ab35359d9674907b32b323042d75de3ef3586bfd482b570749b15eb24a6d68", "anchors": [{"sourceId": "39119eca980f5110cc661428cd067053d2696d78b6f8350ab411acedb843610e", "type": "BTCOpReturn"}], "targetHash": "6502078d48cd734717e77fd672e7216e4cb5e8b21ddf1adf44bbb10e9750b21f", "type": "ChainpointSHA256v2"}}
```
Comment (bedinotti) [inline cert-walletTests/Resources/sample_signed_cert-valid-1.2.0.json:1]: Same question here. It looks like the cert is modified, but the signature isn't, so this should be yet another tampered-with cert that fails validation, right?

**Review by bedinotti — APPROVED** [review]
This looks great! Thanks for doing this. 

And yeah, you shouldn't need to change anything in `cert-web-component`. That's really a proof-of-concept that isn't built for more general parsing yet. Once this model is stable, I'll port the cert parsing stuff to Javascript.

File: cert-walletTests/Resources/sample_signed_cert-1.2.0.json (hunk 2 above)
Comment (kimdhamilton) [reply cert-walletTests/Resources/sample_signed_cert-1.2.0.json:17]: actually...this passed due to issue #14. I intend to add more real samples after issuing some, and will update these samples before we address #14

**Pull request merged**